### DISCUSSION TITLE
Why label is throwing Invalid constant value?

### DISCUSSION THREAD

**Discussion by subhendukundu** [flutter-form-builder-ecosystem/form_builder_image_picker#53] [General]
I have a dynamic form which throwing error on label, not sure why and how to fix it.
```
FormBuilderImagePicker(
    name: field.fieldName,
    decoration: const InputDecoration(
        labelText: label, // throwing Invalid constant value.dart(invalid_constant)
    ),
   maxImages: field.typeConfig?['max_Images'] ?? 1,);
```

**Comment by deandreamatias** [comment]
Remove the const keyword before InputDecoration. If label isn't const, can't use const for widget

**Reply by subhendukundu** [reply]
Ahh I realised that an hour back while talking to @dhuma1981 how stupid I am, didn’t see the const 🤡
Forgot to comment here that. Thanks

**Reply by deandreamatias** [reply]
It's ok, happens to all devs in some moment hahaha

**Reply by subhendukundu** [reply]
I tagged the wrong one, I meant @dhuma1981
